modules/kubernetes-addons/cert-manager/values.yaml

```diff
@@ -1,5 +1,5 @@
 extraArgs:
-  - --enable-certificate-owner-ref=true
```

Comment:
while I agree that the change is valid - I think the better solution might be to just simply remove the extaArgs here and let users pass in the args that suit their use case. 

I think I would prefer that we simply remove the `extraArgs` from the default values file here and leave this as an implementation detail for users. In fact, we can remove the default values file entirely and just set `installCRDs: true` in `set_values` like https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/modules/kubernetes-addons/aws-coredns/main.tf#L59-L64

@vara-bonthu  / @askulkarni2  thoughts?